lib/tree-view.coffee

```diff
@@ -205,19 +219,19 @@ class TreeView extends View
       @show()
 
   entryClicked: (e) ->
-    entry = e.currentTarget
+    entry = e.target.closest('.entry')
     isRecursive = e.altKey or false
     @selectEntry(entry)
-    if entry instanceof DirectoryView
+    if entry.classList.contains('directory')
```

Comment:
Haha, my pleasure! I'm running latest changes through Mocha over and again, so it's easy to spot when something breaks. =)